### PR TITLE
[ISSUE #2061] enhance test cases

### LINKED ISSUE #2061: [Enhancement] enhance test cases
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

enhance event-mesh runtime test cases

### Describe the solution you'd like

enhance event-mesh runtime test cases

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by moremind**
Fixes #2061 

### Motivation

*Explain the content here.*
*Explain why you want to make the changes and what problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/HttpResponseUtils.java [hunk 1]
```diff
@@ -29,37 +27,38 @@
 import io.netty.handler.codec.http.HttpResponse;
 import io.netty.handler.codec.http.HttpResponseStatus;
 import io.netty.handler.codec.http.HttpVersion;
+import org.apache.eventmesh.common.Constants;
```
Comment (xwm1992) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/HttpResponseUtils.java:30]: please don't change this import order

**Review by xwm1992 — CHANGES_REQUESTED** [review]

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/HttpResponseUtils.java (hunk 1 above)
Comment (moremind) [reply eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/HttpResponseUtils.java:30]: I think it must optimize

Comment (xwm1992) [reply eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/HttpResponseUtils.java:30]: but this not followed our project checkstyle config, the ci check won't be succeeded.

Comment (moremind) [reply eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/HttpResponseUtils.java:30]: I think eventmesh need optimize or refactor some nonstandard code.

Comment (xwm1992) [reply eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/HttpResponseUtils.java:30]: I'm not clearly get what you mean, why you change the package import order ?

**Review by xwm1992 — APPROVED** [review]
LGTM

**Pull request merged**